extensions/objects/templates/unicode-string-editor.component.ts

```diff
@@ -23,18 +23,31 @@ import { Component, EventEmitter, Input, OnChanges, OnDestroy, OnInit, Output, S
 import { downgradeComponent } from '@angular/upgrade/static';
 import { Subscription } from 'rxjs';
 import { ExternalSaveService } from 'services/external-save.service';
+
+type UnicodeStringEditorArguments = {
+  currentValue?: {
+    largeInput: string
+  },
+  previousValue?: {
+    largeInput: string
```

Comment:
Why are these optional ?

Reply:
Again, referring to [this](https://github.com/oppia/oppia/pull/13598/files#diff-e681d211f31834396c3bb234d2d42ef9aadc4d75d746ab5847c352c9a9167a6cR84-R91). It is possible that in the form the current and previous values have not been specified.